BangazonScrumptiousJellyfish/BangazonScrumptiousJellyfish/Models/Employee.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonScrumptiousJellyfish.Models
+{
+    public class Employee
+    {
+        
+        public int EmployeeId { get; set; }
+
+        [Required]
+        [Display(Name = "First Name")]
+        public string FirstName { get; set; }
+
+        [Required]
+        [Display(Name = "Last Name")]
+        public string LastName { get; set; }
+
+        [Required]
+        public bool Supervisor { get; set; }
+
+        [Required]
+        public string Email { get; set; }
+
+        [Display(Name = "Employee Name")]
+        public string FullName
+        {
+            get
+            {
+                return $"{FirstName} {LastName}";
+            }
+        }
+
+        public int DepartmentId { get; set; }
```

Comment:
Do you not also need the ComputerId to correctly use this model? (Genuine question - not a request for changes)

Reply:
Current computer Id I mean?